config/jest/server/start.mjs

```diff
@@ -8,11 +9,15 @@ const { PORT = configPaths.ports.test } = process.env
  * Start web server
  */
 export default async function serverStart () {
-  const [server] = await getServers()
+  const servers = await getServers()
 
-  // Server already running
-  if (server) {
-    return
+  // Connection timeout
+  let timeout = 1000
+
+  // Server already running? Wait for exit
+  if (servers.some(({ signalCode }) => signalCode === 'SIGTERM')) {
+    await serverStop()
+    timeout = 0
```

Comment:
We don't need to wait for the port to open when we know it's just closed

Reply:
That's probably worth putting a reminder in the code for the future

```suggestion
    // We don't need to wait for the port to open when we know it's just closed
    timeout = 0
```

Reply:
I've put the comments together like this:

```mjs
await serverStop() // Wait for server to stop
timeout = 0 // No need to wait for start
```